doc/source/whatsnew/v0.25.0.rst

```diff
@@ -158,29 +158,64 @@ cause a ``SparseSeries`` or ``SparseDataFrame`` to be returned, as before.
 Increased minimum versions for dependencies
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Due to dropping support for Python 2.7, a number of optional dependencies have updated minimum versions.
-Independently, some minimum supported versions of dependencies were updated (:issue:`23519`, :issue:`24942`).
-If installed, we now require:
-
-+-----------------+-----------------+----------+
-| Package         | Minimum Version | Required |
-+=================+=================+==========+
-| beautifulsoup4  | 4.4.1           |          |
-+-----------------+-----------------+----------+
-| openpyxl        | 2.4.0           |          |
-+-----------------+-----------------+----------+
-| pymysql         | 0.7.9           |          |
-+-----------------+-----------------+----------+
-| pytz            | 2015.4          |          |
-+-----------------+-----------------+----------+
-| sqlalchemy      | 1.1.4           |          |
-+-----------------+-----------------+----------+
-| xlsxwriter      | 0.7.7           |          |
-+-----------------+-----------------+----------+
-| xlwt            | 1.0.0           |          |
-+-----------------+-----------------+----------+
-| pytest (dev)    | 4.0.2           |          |
-+-----------------+-----------------+----------+
+Due to dropping support for Python 2.7, a number of optional dependencies have updated minimum versions (issue:`25725`, :issue:`24942`, :issue:`25752`).
+Independently, some minimum supported versions of dependencies were updated (:issue:`23519`, :issue:`25554`).
+
+Below is an overview of the minimal versions, as required resp. recommended. Optional libraries below the lowest tested version may still work, but are not considered supported.
+
++-----------------+-----------------+-----------------------+
+| Package         | Minimum Version | Comment               |
```

Comment:
this is too verbose. pls separate into 2 tables, the basic deps (numpy, pytz, bottleneck, numexpr); make the table like we had it exactly

you can make a table for other things, but i am going to limit that list.

Reply:
I kept pytest due to #23519.

I kept the structure of the table because the distinction between the kind of requirement (hard requirement vs. requirement if installed) is clearer like this, and fits with the other table.